here-positioning-android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
```

Comment:
Please Update the wrapper to Gradle version 4.2.1 to align with existing examples.

Reply:
And, here we go again.. https://github.com/heremaps/here-android-sdk-examples/blob/master/advanced-navigation-android/gradle/wrapper/gradle-wrapper.properties

Consistency?